keyboards/decent/tkl/keyboard.json

```diff
@@ -0,0 +1,302 @@
+{
+    "manufacturer": "Bertrand Le roy",
+    "keyboard_name": "DecenTKL",
+    "maintainer": "bleroy",
+    "url": "https://github.com/bleroy/3d-junkyard/blob/main/DecenTKL/",
+    "tags": ["ansi", "tkl", "rgb", "oled", "encoder", "atari"],
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "encoder": true,
+        "bootmagic": true,
+        "rgb_matrix": true,
+        "oled": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9"],
+        "rows": ["GP11", "GP12", "GP13", "GP14", "GP15", "GP16", "GP17", "GP18", "GP19", "GP20"]
+    },
+    "bootmagic": {
+        "matrix": [0, 6]
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "oled": {
+        "driver": "ssd1306",
+        "transport": "i2c"
+    },
```

Comment:
This isnt a thing, plus it would only be duplicating the defaults anyway.
```suggestion
```

Reply:
Removed